docs/content/3.customization.md

```diff
@@ -61,6 +61,31 @@ export default defineAppConfig({
 })
 ```
 
+### Contact us section
+
+Toolkit has the set of default contacts. 
+If you need to use project-specific contacts, specify items with a following syntax to `contacts` array at `app.config.ts` file of your docs website:
+
+```typescript
+export default defineAppConfig({
+  exactproDocs: {
+    contacts: [
+      {
+        contact: 'info@exactpro.com',
+        type: 'email'
+      },
+      {
+        contact: '+4402033191644',
+        type: 'phone'
+      }
+    ]
+  }
+})
```

Comment:
Code block is not closed. That's why it is not recommended to suggest changes to code blocks in markdown.

Please check if `typescript` syntax is highlighted correctly. I'm only sure about `ts` syntax.

Reply:
Yes it is, but I've changed it to `ts`, just to make it uniform